apps/minifront/src/components/ibc/layout.tsx

```diff
@@ -8,12 +8,12 @@ export const IbcLayout = () => {
     <>
       <div className="fixed inset-0 z-[-100] size-full bg-[url('penumbra-logo.svg')] bg-[length:160vmax] bg-fixed bg-[top_50%_left_25vw] bg-no-repeat" />
       <div className='flex flex-1 flex-col gap-4 md:flex-row md:place-content-around'>
-        <Card light className='relative overflow-visible md:self-start'>
+        <Card light className='relative z-10 overflow-visible md:self-start'>
           <LongArrowIcon
             direction='right'
             // Negative calculated margin giving lint issue
             /* eslint-disable-next-line tailwindcss/enforces-negative-arbitrary-values */
-            className='invisible absolute inset-y-0 right-0 my-auto -mr-[calc(30vw-3px)] size-[30vw] text-stone-300 md:visible'
+            className='invisible absolute -top-44 right-0 z-0 -mr-[calc(30vw-3px)] size-[30vw] text-stone-300 md:visible'
```

Comment:
Relocated the arrows to be fixed versus floating in the middle of the card. That way, as the card changes size the arrows don't move.